debian/changelog

```diff
@@ -4,7 +4,25 @@ cloud-init (25.1.2-0ubuntu0~25.04.1) plucky; urgency=medium
     List of changes from upstream can be found at
     https://raw.githubusercontent.com/canonical/cloud-init/25.1.2/ChangeLog
 
- -- James Falcon <james.falcon@canonical.com>  Fri, 02 May 2025 13:02:02 -0500
+ -- James Falcon <james.falcon@canonical.com>  Mon, 19 May 2025 14:20:17 -0500
+
+cloud-init (25.1.1-0ubuntu2) plucky; urgency=medium
+
+  * d/control: Fix how cloud-init-base overwrites cloud-init files.
+    (LP: #2092333)
+  * d/cloud-init-base.postinst: don't version-gate the migration from using
+    cloud-init-base debconf values over cloud-init.
```

Comment:
This entry wasn't part of the released/uploaded plucky 25.1.1-0ubuntu2

it was this instead:
```
cloud-init (25.1.1-0ubuntu2) plucky; urgency=medium

  * d/control: Fix how cloud-init-base overwrites cloud-init files.
    (LP: #2092333)

 -- James Falcon <james.falcon@canonical.com>  Fri, 28 Mar 2025 16:43:14 -0500
```

I think we should retain the d/changelog that  was originally published to plucky to avoid more deltas from previously published d/changelog entries also, the changeset 

This changelog entry belongs up in the new top-most changelog release entry  since it wasn't originally published as part of 25.1.1-0ubuntu2 anyway